packages/components/stories/Table.js

```diff
@@ -347,10 +420,257 @@ ConnectedTable.propTypes = {
 	columns: PropTypes.array,
 	withHeader: PropTypes.bool,
 	onScroll: PropTypes.func,
+	draggable: PropTypes.bool,
 };
 
 const TableWithDND = dndContext(HTML5Backend)(ConnectedTable);
 
+const nameFilterId = 'name-filter';
+
+function matchName(element, filterParams) {
+	const value = filterParams.value;
+	const name = element.name;
+	return Boolean(name.match(value) || name.toLowerCase().match(value));
+}
+
+function createNameFilter() {
+	return {
+		id: nameFilterId,
+		active: false,
+		params: {
+			value: null,
+		},
+		match: matchName,
+		renderer: StringFilterComponent,
+		rendererProps: {
+			placeHolder: 'Filter...',
+			dockable: true,
+			navbar: true,
+		},
+		className: nameFilterId,
+	};
+}
```

Comment:
Those data look very similar to TestData.js
Can you import them instead ?

Reply:
yes but TestData is not intended to be exported, it is an internal module for test purpose.